src/main/scala/is/hail/expr/ir/functions/UtilFunctions.scala

```diff
@@ -9,118 +10,26 @@ import is.hail.expr.types.coerce
 
 object UtilFunctions extends RegistryFunctions {
 
-  def registerAll() {
-    registerCode("triangle", TInt32(), TInt32()) { case (_, n: Code[Int]) => n * (n + 1) / 2 }
+  def parseBoolean(s: String): Boolean = s.toBoolean
```

Comment:
all of the additions in this file are actual additions; all of the deletions got moved to `ArrayFunctions`. Not sure why the formatting got so messed up.